src/Apps/Artist/Routes/WorksForSale/ArtistWorksForSaleRoute.tsx

```diff
@@ -34,11 +38,26 @@ const ArtistWorksForSaleRoute: React.FC<ArtistWorksForSaleRouteProps> = ({
 
   const total = artist.sidebarAggregations?.counts?.total ?? 0
 
+  const experimentName = "diamond_dry_run_for_horizontal_filters"
+  const variant = useFeatureVariant(experimentName)
+  const variantName = variant?.name || "unknown"
+
+  const { trackFeatureVariant } = useTrackFeatureVariant({
+    experimentName,
+    variantName,
+  })
+
+  useEffect(() => {
+    trackFeatureVariant()
+  })
```

Comment:
```tsx
useEffect(trackFeatureVariant, [trackFeatureVariant])
```

Reply:
(This is going to run on every render, otherwise)

Reply:
Thanks, fixed! 🙏